TESTING.rst

```diff
@@ -310,6 +310,19 @@ You can also run all providers but exclude the providers you would like to skip
     breeze testing tests --test-type "Providers[-amazon,google]"
 
 
+Inspecting docker compose after test commands
+---------------------------------------------
+
+Sometimes you need to inspect docker compose after tests command complete
+(for example when test environment could not be properly set due to
```

Comment:
Nit: I think there is a missing `)ˋ somewhere

Reply:
Was. indeed.